src/components/activity/TimelineItemsPanel.svelte

```diff
@@ -17,54 +19,58 @@
 
 <Panel padBody={false}>
   <svelte:fragment slot="header">
-    <GridMenu {gridSection} title="Activity & Resource Types" />
+    <GridMenu {gridSection} title="Activity, Resource, External Event Types" />
   </svelte:fragment>
 
   <svelte:fragment slot="body">
-    <Tabs class="activity-resource-tabs" tabListClassName="activity-resource-tabs-list">
+    <Tabs class="activity-resource-external-event-tabs" tabListClassName="activity-resource-external-event-tabs-list">
```

Comment:
We need to design a better solution for this tab area – don't want there to be some wrapping tab text on the default view for every aerie plan. Could just say "Events" for now?
<img width="351" alt="image" src="https://github.com/user-attachments/assets/352c17b2-cc2e-44a5-ba11-c481b3f59d56">

Reply:
Switched to just events - I think that makes sense for the UI, though I know we were avoiding *not* being explicit about 'External' events other places b/c of the overlap with the 'Events' on the back-end.